src/pages/backOffice/Users/Users.tsx

```diff
@@ -1,8 +1,36 @@
 import * as React from 'react'
 import { UsersProps } from './types'
+import Typography from '@material-ui/core/Typography'
+import UsersTable from '../../../components/Layout/UsersTable'
+import useStyles from './styles'
 
 const Users: React.SFC<UsersProps> = () => {
-  return <h1>This is my Users</h1>
+  const classes = useStyles()
+  const users = [
+    { name: 'Mehmet', surname: 'Baran', birthYear: 1987, birthCity: 63 },
+    {
+      name: 'Zerya Betül',
+      surname: 'Baran',
+      birthYear: 2017,
+      birthCity: 34
+    }
+  ]
+
+  return (
+    <React.Fragment>
+      <Typography
+        component='h1'
```

Comment:
what happens if u remove the component or the variant? there is something weird about that h1 and h5.. u should only keep one of them